drivers/flash/Kconfig.mcux

```diff
@@ -27,6 +27,18 @@ endif # SOC_FLASH_MCUX
 
 if HAS_MCUX_FLEXSPI
 
+config USE_DEDICATED_FLEXSPI_FLASH_DRIVER
+	bool "Use a dedicated flexspi flash driver"
+	help
+	  Enables choosing a flexspi flash driver.
+
+if USE_DEDICATED_FLEXSPI_FLASH_DRIVER
+
```

Comment:
```suggestion
```

Related to the `USE_DEDICATED_FLEXSPI_FLASH_DRIVER` comment.

Reply:
USE_DEDICATED_FLEXSPI_FLASH_DRIVER was made obsolete.